rebasehelper/cli.py

```diff
@@ -133,6 +133,18 @@ def run():
                     os.chdir(repo_path)
                 except OSError:
                     raise RebaseHelperError('Could not change directory to the cloned repository')
+
+            macros = {}
+            for macro in config.rpmmacros:
+                try:
+                    macro = macro.strip()
```

Comment:
Why strip?

Reply:
I think it is safer to remove the leading and trailing whitespaces, not sure how rpm handles them

Reply:
What if the whitespaces are there on purpose? It's unlikely, but I don't think there is a reason to do what's rpm's job.

Reply:
Ok, but <pre>'macro  expression'</pre> won't work with `split()` either. And both work with `rpm -D`.

Reply:
I implemented the parsing with a regex instead. That should be more error-prone. It also covers the 2 cases we mentioned. Although there are could be some advanced cases that won't work with this implementation (but work with `rpm -D`), I think this should suffice.

Reply:
You mean less error-prone, I hope :-)

Reply:
Oh, obviously :D